### PR TITLE
Mark that homejwu

### PR DESCRIPTION
**Pull request by jwu90**
constraints for home page fixed and list view skeleton added, next step is to fix folders and connect the database to the listview

### CODE REVIEW

File: app/src/main/res/layout/activity_home.xml [hunk 1]
```diff
@@ -4,152 +4,226 @@
...
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".Home">
+    tools:context=".Home"
+    tools:layout_editor_absoluteY="81dp">
 
     <SearchView
-        android:id="@+id/searchView2"
-        android:layout_width="52dp"
+        android:id="@+id/search_Recordingbutton"
```
Comment (jstnzl) [inline app/src/main/res/layout/activity_home.xml]: Choose one or the other, either camelCase or snake_case
Every word after the first should be capitalized so searchRecordingButton

File: app/src/main/res/layout/activity_home.xml [hunk 2]
```diff
@@ -4,152 +4,226 @@
...
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.784"
+        app:layout_constraintStart_toEndOf="@+id/hometitletext"
         app:layout_constraintTop_toTopOf="parent"
-        tools:layout_editor_absoluteX="317dp" />
+        app:layout_constraintVertical_bias="0.007" />
 
     <TextView
-        android:id="@+id/textView2"
+        android:id="@+id/hometitletext"
```
Comment (jstnzl) [inline app/src/main/res/layout/activity_home.xml]: Same thing here homeTitleText

File: app/src/main/res/layout/activity_home.xml (hunk 1 above)
Comment (jstnzl) [reply app/src/main/res/layout/activity_home.xml]: Its the easiest to change in the design XML at the top right where it says ID

File: app/src/main/res/layout/activity_home.xml [hunk 3]
```diff
@@ -4,152 +4,226 @@
...
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.227"
+        app:layout_constraintStart_toEndOf="@+id/home_title_dropdown"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.0" />
 
     <Spinner
-        android:id="@+id/spinner"
+        android:id="@+id/home_title_dropdown"
```
Comment (jstnzl) [inline app/src/main/res/layout/activity_home.xml:45]: choose one and use it for the entire page
home_title_dropdown
homeTitleDropdown

File: app/src/main/res/layout/activity_home.xml [hunk 4]
```diff
@@ -4,152 +4,226 @@
...
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.003"
+        app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        tools:layout_editor_absoluteX="16dp" />
+        app:layout_constraintVertical_bias="0.014" />
 
     <View
         android:id="@+id/view"
```
Comment (jstnzl) [inline app/src/main/res/layout/activity_home.xml]: Change the id to something so I know what im looking at

File: app/src/main/res/layout/activity_home.xml [hunk 5]
```diff
@@ -4,152 +4,226 @@
...
         android:background="@android:color/darker_gray"
-        tools:layout_editor_absoluteX="0dp"
-        tools:layout_editor_absoluteY="72dp" />
+        app:layout_constraintBottom_toTopOf="@+id/add_folder_btn"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/search_Recordingbutton" />
 
     <View
         android:id="@+id/view3"
```
Comment (jstnzl) [inline app/src/main/res/layout/activity_home.xml]: change id so I know what view this is for

File: app/src/main/res/layout/activity_home.xml [hunk 6]
```diff
@@ -4,152 +4,226 @@
...
-        tools:layout_editor_absoluteY="309dp" />
+        app:layout_constraintBottom_toTopOf="@+id/floatingActionButton2"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/see_more_text"
+        app:layout_constraintVertical_bias="0.0" />
 
     <View
         android:id="@+id/view2"
```
Comment (jstnzl) [inline app/src/main/res/layout/activity_home.xml]: same here

File: app/src/main/res/layout/activity_home.xml [hunk 7]
```diff
@@ -4,152 +4,226 @@
...
 
     <ImageView
-        android:id="@+id/imageView"
-        android:layout_width="107dp"
-        android:layout_height="62dp"
-        android:layout_marginTop="40dp"
-        app:layout_constraintTop_toBottomOf="@+id/textView3"
-        app:srcCompat="@drawable/folder_bg4"
-        tools:layout_editor_absoluteX="7dp" />
+        android:id="@+id/folder1"
```
Comment (jstnzl) [inline app/src/main/res/layout/activity_home.xml:134]: folder1 is fine for now but if you know what its going to be for then change the name

File: app/src/main/res/layout/activity_home.xml [hunk 8]
```diff
@@ -4,152 +4,226 @@
...
+        app:layout_constraintHorizontal_bias="0.03"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.225"
+        app:srcCompat="@drawable/folder_bg4" />
 
     <ImageView
-        android:id="@+id/imageView2"
-        android:layout_width="101dp"
+        android:id="@+id/folder2"
```
Comment (jstnzl) [inline app/src/main/res/layout/activity_home.xml:150]: same goes for this, and folder 3

File: app/src/main/res/layout/activity_home.xml [hunk 9]
```diff
@@ -4,152 +4,226 @@
...
+        android:textSize="23sp"
+        app:layout_constraintBottom_toTopOf="@+id/home_title_dropdown"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.01"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.163" />
 
     <android.support.design.widget.FloatingActionButton
         android:id="@+id/floatingActionButton2"
```
Comment (jstnzl) [inline app/src/main/res/layout/activity_home.xml]: what does this button do? the id doesnt tell me anything
also it looks weird idk if intentional- theres also some white around the edges that I can see did you use paint or something to rm the bg
image

**Review by jstnzl — CHANGES_REQUESTED** [review]
finally i see what the home page is supposed to look like nice
changes look good, see comments about naming convention and change all the components to fit
Maybe reference record.xml to see how duke did it

File: app/src/main/res/layout/activity_home.xml (hunk 1 above)
Comment (jwu90) [reply app/src/main/res/layout/activity_home.xml]: fixed

File: app/src/main/res/layout/activity_home.xml (hunk 2 above)
Comment (jwu90) [reply app/src/main/res/layout/activity_home.xml]: fixed

File: app/src/main/res/layout/activity_home.xml (hunk 3 above)
Comment (jwu90) [reply app/src/main/res/layout/activity_home.xml:45]: fixed

File: app/src/main/res/layout/activity_home.xml (hunk 4 above)
Comment (jwu90) [reply app/src/main/res/layout/activity_home.xml]: changed to hr1, hr2 ,hr3 after html horizontal line <hr>

File: app/src/main/res/layout/activity_home.xml (hunk 9 above)
Comment (jwu90) [reply app/src/main/res/layout/activity_home.xml]: this button was downloaded straight from duke's google drive and uploaded.

File: app/src/main/res/layout/activity_home.xml [hunk 10]
```diff
@@ -4,152 +4,226 @@
...
 
     <android.support.design.widget.FloatingActionButton
-        android:id="@+id/floatingActionButton2"
-        android:layout_width="89dp"
-        android:layout_height="79dp"
-        android:layout_marginStart="159dp"
-        android:layout_marginTop="149dp"
-        android:layout_marginEnd="16dp"
-        android:layout_marginBottom="27dp"
+        android:id="@+id/floating_action_button2"
```
Comment (jstnzl) [inline app/src/main/res/layout/activity_home.xml]: name?

**Review by jstnzl — CHANGES_REQUESTED** [review]

**Pull request merged**